### PR TITLE
Add test cases for JSON resource marshaling

### PR DESCRIPTION
**Pull request by sagar23sj**
Added test cases for following change events :
* ProjectChange
* ProjectCardChange
* ProjectColumnChange

Relates to : #55

### CODE REVIEW

File: github/event_types_test.go [hunk 1]
```diff
@@ -103,3 +103,94 @@ func TestEditChange_Marshal_BaseChange(t *testing.T) {
...
+	}
+
+	want := `{
+		"name": {
+			"from": "NameFrom"
+		  }
+	}`
+
+	testJSONMarshal(t, u, want)
+
```
Comment (gmlewis) [inline github/event_types_test.go:195]: Let's please delete this blank line for consistency with the other tests.

Comment (sagar23sj) [reply github/event_types_test.go:195]: Sure @gmlewis. I will fix this right away.

**Review by gmlewis — APPROVED** [review]
Thank you, @sagar23sj !
LGTM.

Awaiting second LGTM before meging.

**Comment by gmlewis** [comment]
Friendly ping @wesleimp

**Comment by sagar23sj** [comment]
Seems like, this will also break because of changes in #1900 @gmlewis

**Review by Parker77 — APPROVED** [review]
LGTM.

**Comment by gmlewis** [comment]
Hmmm... the tests still pass.  Merging.

**Comment by gmlewis** [comment]
Although after merging, tests fail... so I'm not sure that GitHub was actually testing master.  I'll create a PR.

**Comment by sagar23sj** [comment]
Thank You @gmlewis. I will fix the other PRs affected by #1900.

**Pull request merged**